### ISSUE TITLE
[build] error while loading shared libraries: librarylib.so.301...

### ISSUE THREAD

**Issue by KingMagnaRool** [raysan5/raylib#1424]
### Issue description

I installed Raylib using the recommended CMAKE guide on the website.  I can compile an executable with the flags "-lraylib -lGL -lm -lpthread -ldl -lrt -lX11" either in the terminal or in Code::Blocks, but I couldn't run it as it complained that there was a missing library "librarylib.so.301."  This happened on three separate computers.  I knew that by the third time that the aforementioned file was the culprit, as I fixed the issue by deleting it in /usr/local/lib. 

### Environment

One of the computers was running Pop OS 20.04, the other two were running Kubuntu 20.04.  All of them are x86 machines; the GPU is certainly not the issue, neither is the OpenGL as I was able to fix the issue.

### Issue Screenshot

![raylibissue_1_1](https://user-images.githubusercontent.com/42488667/97947349-0949e800-1d5b-11eb-9e5d-5f688901797a.png)
![raylibissue_1_2](https://user-images.githubusercontent.com/42488667/97947357-0d760580-1d5b-11eb-9b85-965f2264eddc.png)
![raylibissue_1_3](https://user-images.githubusercontent.com/42488667/97947363-0fd85f80-1d5b-11eb-8e53-53df610aea79.png)
![raylibissue_1_4](https://user-images.githubusercontent.com/42488667/97947371-123ab980-1d5b-11eb-9ad1-c7ce69634b8f.png)

### Code Example

#include "raylib.h"

**Comment by chriscamacho** [comment]
did you run ldconfig ?

**Comment by KingMagnaRool** [comment]
I did not run Idconfig, but that wasn't in the

**Comment by KingMagnaRool** [comment]
Didn't mean to close the issue

No I didn't, but that wasn't in the raylib install guide on Linux, at least for cmake.  I ran exactly what was recommended.
"git clone https://github.com/raysan5/raylib.git raylib
cd raylib
mkdir build && cd build
cmake -DSHARED=ON -DSTATIC=ON ..
make
(sudo) make install"
The issue was librarylib.so.301, since when I deleted that as shown in the pictures, I could run a raylib program.  I could compile one without deleting that file, but for whatever reason that file prevented me from running it.  I don't know if Idconfig would change that, but I followed exactly what was on the install guide.
